db/seeds.rb

```diff
@@ -71,9 +71,7 @@
 
     user = User.create!(
       name: name,
-      summary: Faker::Lorem.paragraph_by_chars(number: 199, supplemental: false),
```

Comment:
With delegation to the profile fields removed we should now set this and `website_url` on the profile directly. The e2e seeds had already been updated by someone else.